aws/resource_aws_spot_fleet_request.go

```diff
@@ -311,7 +313,94 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				},
 				Set: hashLaunchSpecification,
 			},
-			// Everything on a spot fleet is ForceNew except target_capacity
+			"launch_template_configs": {
+				Type:          schema.TypeSet,
```

Comment:
Can you please add a test that declares multiple `launch_template_config`? Thank you!